code/modules/projectiles/projectile/bullets/rifle.dm

```diff
@@ -61,17 +61,119 @@
 	embedding = list(embed_chance=60, fall_chance=2, jostle_chance=2, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=3, jostle_pain_mult=2, rip_time=10)
 	embed_falloff_tile = -5
 	wound_falloff_tile = -2
-	shrapnel_type = /obj/item/stack/rods
+	shrapnel_type = /obj/item/ammo_casing/rebar
+
+/obj/projectile/bullet/rebar/proc/handle_drop(datum/source, obj/item/ammo_casing/rebar/newcasing)
 
 /obj/projectile/bullet/rebarsyndie
 	name = "rebar"
 	icon_state = "rebar"
-	damage = 35
+	damage = 55
 	speed = 0.4
 	dismemberment = 2 //It's a budget sniper rifle.
 	armour_penetration = 20 //A bit better versus armor. Gets past anti laser armor or a vest, but doesnt wound proc on sec armor.
 	wound_bonus = 10
-	bare_wound_bonus = 10
+	bare_wound_bonus = 20
 	embedding = list(embed_chance=80, fall_chance=1, jostle_chance=3, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=3, jostle_pain_mult=2, rip_time=14)
 	embed_falloff_tile = -3
-	shrapnel_type = /obj/item/stack/rods
+	shrapnel_type = /obj/item/ammo_casing/rebar/syndie
+
+/obj/projectile/bullet/rebar/zaukerite
+	name = "zaukerite shard"
+	icon_state = "rebar"
+	damage = 60
+	speed = 0.4
+	dismemberment = 2
+	damage_type = TOX
+	eyeblur = 5
+	armour_penetration = 15 // not nearly as good, as its not as sharp.
+	wound_bonus = 0
+	bare_wound_bonus = 20
+	embedding = list(embed_chance=90, fall_chance=0, jostle_chance=5, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.8, pain_mult=6, jostle_pain_mult=2, rip_time=30)
+	embed_falloff_tile = -2 // very spiky.
+	shrapnel_type = /obj/item/ammo_casing/rebar/zaukerite
+
+/obj/projectile/bullet/rebar/hydrogen
+	name = "metallic hydrogen bolt"
+	icon_state = "rebar"
+	damage = 40
+	speed = 0.6
+	dismemberment = 0 //goes through clean.
+	damage_type = BRUTE
+	armour_penetration = 30 //very pointy.
+	projectile_piercing = PASSMOB //felt this might have been a nice compromise for the lower damage for the difficulty of getting it
+	wound_bonus = -15
+	bare_wound_bonus = 10
+	embedding = list("embed_chance" = 50, "fall_chance" = 2, "jostle_chance" = 3, "ignore_throwspeed_threshold" = TRUE, "pain_stam_pct" = 0.6, "pain_mult" = 4, "jostle_pain_mult" = 2, "rip_time" =18)
+	embed_falloff_tile = -3
+	shrapnel_type = /obj/item/ammo_casing/rebar/hydrogen
+
+/obj/projectile/bullet/rebar/healium
+	name = "healium bolt"
+	icon_state = "rebar"
+	damage = 0
+	speed = 0.4
+	dismemberment = 0
+	damage_type = BRUTE
+	armour_penetration = 100
+	wound_bonus = -100
+	bare_wound_bonus = -100
+	embedding = list(embed_chance = 0)
+	embed_falloff_tile = -3
+	shrapnel_type = /obj/item/ammo_casing/rebar/healium
+
+/obj/projectile/bullet/rebar/healium/on_hit(atom/target, blocked = 0, pierce_hit)
+	. = ..()
+	if(!isliving(target))
+		return BULLET_ACT_HIT
+	var/mob/living/breather = target
+	breather.SetSleeping(3 SECONDS)
```

Comment:
i'd recommend adding some sort of counter to the sleeping from this otherwise it's basically a one-hit kill as long as the person wielding the crossbow has any other weapon.

Reply:
> i'd recommend adding some sort of counter to the sleeping from this otherwise it's basically a one-hit kill as long as the person wielding the crossbow has any other weapon. i absolutely dread seeing the stunbaton being used with this

I thought about this, and while yeah its a ranged stun, there are easier methods of doing it (e.g. fake beer syringe gun/dart shells, the chem version of healium, the normal traitor mini-crossbow) that dont require healium synth to get, and also don't heal who you're shooting at. 

If it gets abused really hard I might change it, but I dont see people who are seeking to powergame real hard learning healium synth to get a 3 second sleep stun.

Reply:
Also of note: Nitrium, also from atmos, makes you immune to sleeps if you're breathing it.